src/browser/components/MainPage.jsx

```diff
@@ -474,13 +477,11 @@ export default class MainPage extends React.Component {
           });
         }}
         onSave={(newTeam) => {
-          this.props.teams.push(newTeam);
           this.setState({
             showNewTeamModal: false,
-            key: this.props.teams.length - 1,
```

Comment:
Team length can be 0, in which case key will be -1, and that view will be hidden because it will be considered !isActive